pages/common/xev.md

```diff
@@ -0,0 +1,20 @@
+# xev
+
+> Print contents of X events.
+> More information: <https://gitlab.freedesktop.org/xorg/app/xev>.
+
+- View all occuring X events:
+
+`xev`
+
+- Monitor the X events of the root window instead of creating a new one:
+
+`xev --root`
+
+- View the X events of a particular window:
+
+`xev -id {{Window_id}}`
+
+- View a category of X events (multiple flags can be chained):
```

Comment:
First time reading this, I needed a minute to parse the sentence properly. Not sure, how to phrase it properly though. This would be ok, I think:

```suggestion
- View X events from a given category (multiple flags can be chained):
```

Reply:
Yea you're correct, that is easier to comprehend.